hyprctl/main.cpp

```diff
@@ -22,37 +22,54 @@
 #include <filesystem>
 #include <stdarg.h>
 
-const std::string USAGE = R"#(usage: hyprctl [(opt)flags] [command] [(opt)args]
-
-commands:
-    monitors
-    workspaces
-    activeworkspace
-    clients
-    activewindow
-    layers
-    devices
-    binds
-    dispatch
-    keyword
-    version
-    kill
-    splash
-    hyprpaper
-    reload
-    setcursor
-    getoption
-    cursorpos
-    switchxkblayout
-    seterror
-    setprop
-    plugin
-    notify
-    globalshortcuts
-    instances
-
-flags:
+const std::string USAGE = R"#(usage:  hyprctl [flags] [<command> [args]]
+        hyprctl --batch {<command 1> [args] ; <command 2> [args] ; ...}
+
+LISTING COMMANDS:
+    monitors:           List outputs
```

Comment:
why are the `:` everywhere?

Reply:
They probably took they style of writing from their language